Formula/grass.rb

```diff
@@ -70,6 +63,7 @@ def install
       "--with-sqlite",
       "--with-odbc",
       "--with-geos=#{Formula["geos"].opt_bin}/geos-config",
+      "--with-proj-share=#{Formula["proj"].opt_share}/proj",
```

Comment:
This should be `#{Formula["proj"].opt_pkgshare}` instead. Also, am I overlooking something because I don't see `proj` among the dependencies of this formula?